scripts/install-rkt.sh

```diff
@@ -29,6 +29,16 @@ tar xvzf rkt-v"${version}".tar.gz
 cat <<EOF >install-pak
 #!/bin/bash
 
+# fix mkdir issues with checkinstall and fstrans
+for dir in /usr/lib/rkt/stage1-images/\\
+        /usr/share/man/man1/\\
+        /usr/share/bash-completion/completions/\\
+        /usr/lib/tmpfiles.d/\\
+        /usr/lib/systemd/system/
+do
+    mkdir -p \$dir 2>/dev/null || :
```

Comment:
You rigth with both points. `mkdir -p` does not abort the script since there is no `-e`. I still prefer to handle known/expected failures when writing scripts since the next guy debugging the code will not stumble upon the known behavior while trying to figure out something else.

I'd be in favor of adding `-e` to the shebang. I can provide the PR that cleans this up.